src/Site/SiteAwareTrait.php

```diff
@@ -51,23 +51,24 @@ public function getSite($site_id)
      *
      * @TODO This should be moved to the input/validation stage when that is available.
      *
-     * @param string $site_env_id The site/environment id in the form <site>[.<env>]
-     * @param string $default_env The default environment to use if none is specified.
-     * @return array The site and environment in an array.
+     * @param string  $site_env_id The site/environment id in the form <site>[.<env>]
+     * @param string  $default_env The default environment to use if none is specified
+     * @param boolean $require_env True to raise an exception if environment is missing
+     * @return array  The site and environment in an array.
      * @throws \Terminus\Exceptions\TerminusException
      */
-    public function getSiteEnv($site_env_id, $default_env = null)
+    public function getSiteEnv($site_env_id, $default_env = null, $require_env = true)
```

Comment:
We didn't have `require_env` before, so setting $require_env to default to true will make every other command using it as it is now throw an error when the env is not included.